boltdesignsystem/bolt
# DS 656 Form Documentation Update

## Jira

https://pegadigitalit.atlassian.net/browse/DS-656

## Summary

Form Component was updated with the standard of the new docs.

## Details

- form inputs variations were updated with the new docs
- places with old form component usage were replaced with the new usage
- old docs were removed
- only `twig` docs examples were created for now

## How to test

Pull the branch. Check if the documentation is useful and there are no regressions.
